tests/v2/validation/snapshot/snapshot_additional_test.go

```diff
@@ -43,36 +43,42 @@ func (s *SnapshotAdditionalTestsTestSuite) SetupSuite() {
 	s.client = client
 }
 
-func (s *SnapshotAdditionalTestsTestSuite) TestSnapshotReplaceWorkerNode() {
-	snapshotRestoreAll := &etcdsnapshot.Config{
+func (s *SnapshotAdditionalTestsTestSuite) TestSnapshotReplaceNodes() {
+	etcdSnapshotRestore := &etcdsnapshot.Config{
 		UpgradeKubernetesVersion: "",
-		SnapshotRestore:          "all",
+		SnapshotRestore:          "none",
```

Comment:
Just confirming, we have coverage of all and k8s version snapshots elsewhere which is why you're removing them from here?

Reply:
Correct, it's a bit excessive to do that here. Additional tests aren't testing each permutation, it's just testing a specific scenario. So I think having `none` is satisfactory here.